dev/META6.json

```diff
@@ -21,7 +21,7 @@
     "Net::SMTP::Client::Async",
     "OO::Monitors",
     "Shell::Command",
-    "Spreadsheet::XLSX:ver<0.3+>",
+    "Spreadsheet::XLSX:ver<0.2+>",
```

Comment:
The new release was `0.2.1`.

Reply:
Oh, should have guessed as it was a bug fix.